go/chat/helper.go

```diff
@@ -233,6 +237,138 @@ func (h *Helper) GetMessages(ctx context.Context, uid gregor1.UID, convID chat1.
 	return messages, nil
 }
 
+func (h *Helper) AckMobileNotificationSuccess(ctx context.Context, pushIDs []string) {
+	defer h.Trace(ctx, func() error { return nil }, "AckMobileNotificationSuccess")()
+	// Get session token
+	nist, _, err := h.G().ActiveDevice.NISTAndUID(ctx)
+	if nist == nil {
+		h.Debug(ctx, "AckMobileNotificationSuccess: got a nil NIST, is the user logged out?")
+		return
+	}
+	if err != nil {
+		h.Debug(ctx, "AckMobileNotificationSuccess: failed to get logged in session: %s", err.Error())
+		return
+	}
+
+	// Make an ad hoc connection to gregor
```

Comment:
Why not use the standard gregor connection?

Reply:
Don't want all the sync stuff to go off, just need the minimal connection to get this RPC out. I could refactor a bunch of stuff in gregor.go to make that happen, or I could just have this special case here.